pycbc/io/hdf.py

```diff
@@ -1028,41 +1028,38 @@ def to_coinc_hdf_object(self, file_name):
         # Some fields are special cases
         logging.info("Outputting search results")
         time = self.get_end_time()
-        ofd.create_dataset('time', data=time, dtype=np.float32)
+        ofd['time'] = time
 
         if self._inclusive:
```

Comment:
would it be neater to roll both the `if self._inclusive` switches into one?

Reply:
fair, will do